test/docker/networktest/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# escape=`
+FROM microsoft/windowsservercore
+
+SHELL ["powershell.exe","-command"]
+
+# the source msi should change on a daily basis
+# the destination should not change
+ADD PSCore.msi /PSCore.msi
```

Comment:
this is done with the New-DockerTestBuild.ps1 script. I broke it out to it's own function.